carbon-design-system/carbon
# fix(storybook): use plex in manager ui even when not installed

Something I noticed in #16910 - on machines that do not have Plex installed, the storybook uses a system default font. 

#### Changelog

**Changed**

- Update storybook's `manager-head.html` to load plex from the CDN into the [Manager](https://storybook.js.org/docs/addons#storybook-basics) portion of storybook (sidebar nav, controls pane, etc.)

#### Testing / Reviewing

- Open the deploy preview on a machine with plex installed
  - inspect in the firefox dev tools 'fonts' pane, the locally installed version of plex should be on the page in addition to the one(s) loaded from the deploy preview itself.
- Open the deploy preview on a machine without plex installed
  - inspect in the firefox dev tools ... you should see plex being loaded from `s81c.com/...`
  - The storybook Manager ui should be rendered with plex

| Windows w/out local Plex (old) | Windows w/out local Plex (this PR) |  Mac w/ local Plex (prod) |
|--------|--------|--------|
| <img width="224" src="https://github.com/user-attachments/assets/a0723ae6-12c9-4788-9f64-b9342706d02f"> | <img width="224" src="https://github.com/user-attachments/assets/711f8ddd-5350-4702-9236-e848dcfe4bb6" /> | <img width="224" src="https://github.com/user-attachments/assets/7ec68c45-85e6-416b-be00-c3d34bb5657e"/> |
